packages/api/config/lambdas.yml

```diff
@@ -99,7 +99,7 @@ jobs:
 CustomBootstrap:
   handler: index.bootstrap
   timeout: 100
-  memory: 256
+  memory: 1024
```

Comment:
What is the reasoning for 1024 as opposed to say, 512?

Reply:
no reason, it's somewhat arbitrary

Reply:
I *think* this affects cost? Maybe? So might be better to keep it on the lower end?

Reply:
right you are! https://aws.amazon.com/lambda/pricing/ good call

Reply:
decreased to `512`